webpack/assets/javascripts/react_app/components/Layout/__tests__/__snapshots__/Layout.test.js.snap

```diff
@@ -53,97 +53,175 @@ exports[`Layout rendering renders layout 1`] = `
+          changeLocation={[Function]}
+          changeOrganization={[Function]}
+          currentLocation="Any Location"
+          currentOrganization="Any Organization"
+          data={
             Object {
-              "href": "/locations/1-yaml/select",
-              "id": 1,
-              "title": "yaml",
-            },
-            Object {
-              "href": "/locations/2-london/select",
-              "id": 2,
-              "title": "london",
-            },
-            Object {
-              "href": "/locations/3-norway/select",
-              "id": 3,
-              "title": "norway",
-            },
-          ]
-        }
-        onLocationClick={[Function]}
-        onOrgClick={[Function]}
-        organizations={
-          Array [
-            Object {
-              "href": "/organizations/1-org1/select",
-              "id": 1,
-              "title": "org1",
-            },
-            Object {
-              "href": "/organizations/2-org2/select",
-              "id": 2,
-              "title": "org2",
-            },
-          ]
-        }
-        taxonomiesBool={
-          Object {
-            "locations": true,
-            "organizations": true,
-          }
-        }
-      />
-      <UserDropdowns
-        changeActiveMenu={[Function]}
-        className=""
-        instanceTitle="Production"
-        notificationUrl="/notification_recipients"
-        stopImpersonationUrl="/users/stop_impersonation"
-        user={
-          Object {
-            "current_user": Object {
-              "firstname": "Admin",
-              "id": 4,
-              "lastname": "User",
-              "login": "admin",
-              "name": "Admin User",
-            },
-            "user_dropdown": Array [
-              Object {
-                "children": Array [
+              "instance_title": "Production",
+              "locations": Object {
+                "available_locations": Array [
+                  Object {
+                    "href": "/locations/1-yaml/select",
+                    "id": 1,
+                    "title": "yaml",
+                  },
+                  Object {
+                    "href": "/locations/2-london/select",
+                    "id": 2,
+                    "title": "london",
+                  },
                   Object {
-                    "isDivider": false,
-                    "onClick": [MockFunction],
-                    "title": "Aa",
+                    "href": "/locations/3-norway/select",
+                    "id": 3,
+                    "title": "norway",
                   },
+                ],
+                "current_location": "london",
+              },
+              "logo": "/assets/header_logo-c9614c16f2ee399ae9cb7f36ec94b9a26bf8cf9eabaa7fe6099bf80d1f7940db.svg",
+              "menu": Array [
+                Object {
+                  "children": Array [
+                    Object {
+                      "exact": true,
```

Comment:
I think 2-3 menu items would be enough for the snapshot, it will reduce the file size